src/Core/Content/Flow/Dispatching/FlowExecutor.php

```diff
@@ -200,6 +204,15 @@ private function callHandle(ActionSequence $sequence, StorableFlow $event): void
             return;
         }
 
+        // We could also make the async flag configurable in the action config modal and check here if it should run async or not.
+        // That way we can also get rid of the shopware.yaml setting.
+        // It would give us the flexibility to run the same action async or sync depending on the flow configuration.
+        if ($this->messageBus !== null && $action instanceof AsyncFlowAction) {
+            $this->messageBus->dispatch(new FlowActionMessage($sequence, $event, $event->getContext()));
```

Comment:
Just trying to understand the need for this and it's differences to this initiative we have already ongoing and already available behind a feature flag:

https://github.com/shopware/shopware/discussions/6750

Specifically the latest proposal to move to the queue:
https://github.com/shopware/shopware/discussions/6750#discussioncomment-12628623

I guess that should already cover the issue as well, it's just missing the implementation change.

@Benedikt-Brunner  feel free to jump in here. Would be nice to know if that change is still on your radar or if we should take over? 🙂